### ISSUE TITLE
Lower Temperature on ironing to avoid heat creep

### ISSUE THREAD

**Issue by bitranox** [supermerill/SuperSlicer#3080]
**Is your feature request related to a problem? Please describe.**
After ironing some surfaces, i can not continue to print, because of heat creep - the filament got stuck,
because there was no extruding for some time.

**Describe the solution you'd like**
As soon as ironing starts, i want to lower the temperature to avoid heat creep - 
and reset to the temperature before when ironing is finished.

**Describe how it would work**
see above

**Describe alternatives you've considered**
alternatively I can write my own postprocessing script

**Additional context**
--

**Comment by PersonalArtWorkshop** [comment]
Yes, I desperately want a solution to the heat creep issue too. 
Maybe even if the fan is turned up for ironing? wouldn't that help?
